test/functional/test_framework/test_node.py

```diff
@@ -549,11 +550,17 @@ def p2p(self):
         assert self.p2ps, self._node_msg("No p2p connection")
         return self.p2ps[0]
 
-    def disconnect_p2ps(self):
+    def num_connected_mininodes(self):
+        """Return number of test framework p2p connections to the node."""
+        return len([peer for peer in self.getpeerinfo() if peer['subver'] == MY_SUBVERSION])
+
+    def disconnect_p2ps(self, wait_for_disconnect=False):
```

Comment:
Also agree with John that this should not be a configurable parameter to keep the complexity down

Reply:
Ok! 👍  will update with the rebase since it seems like the performance issue isn't too bad